util/regexp-assemble/regexp-942130.data

```diff
@@ -0,0 +1,16 @@
+([\s'\"`\(\)]*?)([\d\w]++)([\s'\"`\(\)]*?)=([\s'\"`\(\)]*?)\2
```

Comment:
What's the purpose of the double ++? This looks odd to me.

Reply:
The `++` is a possessive quantifier. It matches the regex between zero and more times, but it doesn't give back the already matched string.

Reply:
Thanks for the cluestick. I learned something today.